src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -148,15 +149,25 @@ protected void transRollback(Connection conn) {
     }
   }
 
-  protected void transCommit(Connection conn) {
-    try {
-      conn.commit();
-    } catch (SQLException se) {
-      LOG.debug(se.getMessage());
-      transRollback(conn);
+  private void performInserts(List<PreparedStatement> stmts, Connection conn) throws Exception {
```

Comment:
nit, prefer the following syntax to avoid the list conversion at the call-site:
```
private void performInserts(Connection conn, PreparedStatement... stmts) throws Exception {
```

Reply:
can we call this something like `performInsertsWithRetry`?

Reply:
Done.